tests/integration/tier0/non-destructive/config-file/test_config_file.py

```diff
@@ -70,47 +113,75 @@ def test_config_cli_priority(convert2rhel):
 
     assert c2r.exitstatus == 1
 
-    remove_files(config)
-
-
-def test_config_standard_paths_priority_diff_methods(convert2rhel):
-    config = [
-        Config("~/.convert2rhel.ini", "[subscription_manager]\npassword = config_password"),
-        Config("/etc/convert2rhel.ini", "[subscription_manager]\nactivation_key = config2_activationkey"),
-    ]
-    create_files(config)
 
+@pytest.mark.parametrize(
+    "set_up_config_files",
+    [
+        [
+            Config(
+                "~/.convert2rhel.ini",
+                """\
+[subscription_manager]
+password = config_password
+                   """,
+            ),
+            Config(
+                "/etc/convert2rhel.ini",
+                """
+[subscription_manager]
+activation_key = config2_activationkey
+""",
+            ),
+        ]
+    ],
+    indirect=True,
+)
+def test_config_standard_paths_priority_diff_methods(convert2rhel, set_up_config_files):
     with convert2rhel("--debug") as c2r:
-        c2r.expect("DEBUG - Found password in /root/.convert2rhel.ini")
-        c2r.expect("DEBUG - Found activation_key in /etc/convert2rhel.ini")
+        c2r.expect("DEBUG - Checking configuration file at /root/.convert2rhel.ini", timeout=120)
+        c2r.expect("DEBUG - Found password in subscription_manager")
+        c2r.expect("DEBUG - Found activation_key in subscription_manager")
+
         c2r.expect(
             "WARNING - Passing the RHSM password or activation key through the --activationkey or --password options"
```

Comment:
It doesn't seem correct to me. The c2r reports "Passing the RHSM password or activation key through the --activationkey or --password options is insecure..." but we aer calling convrt2rhel only with the `--debug` option, @danmyway, @r0x0d?

Reply:
Yeah, it feels off as well but this debug log has been there since forever. 

@kokesak, the ticket got delayed for next version, so I will have more time to revisit it and rework some of those things.